webapp/channels/src/i18n/en.json

```
@@ -2816,19 +2816,25 @@
   "admin.site.localization": "Localization",
   "admin.site.localization.auto_translation.off": "Off",
   "admin.site.localization.auto_translation.on": "On",
+  "admin.site.localization.autoTranslationAgentsError": "Mattermost Agents plugin is either disabled or not configured properly.",
   "admin.site.localization.autoTranslationInfo": "Auto-translation must also be enabled in each channel where it's needed.",
-  "admin.site.localization.autoTranslationInfoSecondary": "When multiple languages are detected, users will be prompted to enable auto-translation. [Learn more](https://docs.mattermost.com/).",
+  "admin.site.localization.autoTranslationLLMConfigNote": "LLMs must first be configured in the Agents plugin.",
+  "admin.site.localization.autoTranslationLLMServiceTitle": "AI Service",
```

Adjust 'service' to lowercase

```suggestion
  "admin.site.localization.autoTranslationLLMServiceTitle": "AI service",
```